src/api.ts

```diff
@@ -3855,6 +3856,159 @@ export const methods = {
       countFailedResponse(api_name, 'exception in /contract/accesslist')
     }
   },
+  shardeum_getNodeList: async function (args: RequestParamsLike, callback: JSONRPCCallbackTypePlain) {
+    const api_name = 'shardeum_getNodeList'
+    const ticket = crypto
+      .createHash('sha1')
+      .update(api_name + Math.random() + Date.now())
+      .digest('hex')
+    logEventEmitter.emit('fn_start', ticket, api_name, performance.now())
+
+    try {
+      if (!Array.isArray(args) || args.length > 1 || (args.length === 1 && typeof args[0] !== 'object')) {
+        callback({ code: -32602, message: 'Invalid params: Expected an object or no parameters.' }, null)
+        countFailedResponse(api_name, 'Invalid params')
+        return
+      }
+
+      const params = args[0] || {}
+      const page = Math.max(1, parseInt(params.page) || 1)
+      const limit = Math.min(1000, Math.max(1, parseInt(params.limit) || 100))
```

Comment:
This is not required. Please change this to 

`const limit = Math.min(1000, parseInt(params.limit) || 100)`